vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
> Hmm, this all seems a bit brittle to me. It also seems unclear how a dataset is supposed to choose the enumeration value (i.e. so that different datasets would be using the same value). For example, what is the difference between city, local, locality, and municipality (and county in the case of a City and County like San Francisco)?

Indeed, I expect we would threat city, local, locality, and municipality as equivalent for collision-detection purposes. City/counties are slightly trickier, in that if one dataset calls SF a city and another calls it a county we'd be at risk for serving duplicate contests. On the bright side, we already have logic to treat aliased OCD IDs as belonging to every applicable district type, so if one of those datasets is BIP (which already uses OCD IDs) we would correctly detect the collision.

> Also, just to be clear, my original suggestion wasn't to invent a new identification scheme, but to introduce a parent object that would serve to group the districts within a numbering sequence. It is agnostic to the particular identification scheme: it doesn't require one but could still be used in conjunction with one (and in conjunction with 1622.2).

That either *is* an identifications scheme, with the slightly awkward extra restriction that different portions of the identifier go in different fields of different XML objects, or it doesn't solve any problems (because we can't tell whether two ElectoralDistricts in different feeds represent the same actual district or not).

Moreover, it is no more likely (read: 0% now, possible in the future) that any EMS will be able to supply this information usefully than that it will be able to supply OCD IDs for each electoral district, and the latter approach totally solves every problem that ElectoralDistrict.type is useful for anyway.